### PR TITLE
Setup a mail server with Mailcow

### PR DESCRIPTION
**Pull request by ntimo**
I wrote a new article on how to setup your own mail server with Mailcow :)

I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Timo 0mfhniozkb9s4q7e6ap8yvlt@nowitzki.me

### CODE REVIEW

File: tutorials/setup-mailserver-with-mailcow/01.en.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+author_img: "https://avatars3.githubusercontent.com/u/6145026"
+author_description: "Docker is awesome"
+header_img: "https://mailcow.email/img/cow_mailcow.svg"
+---
+
+# Installing Mailcow on Ubuntu
+
+## Introduction
+
+In this tutorial you will setup your own Mailserver running on a Ubuntu Server in the Hetzner Cloud.
```
Comment (n-se) [inline tutorials/setup-mailserver-with-mailcow/01.en.md]:
```suggestion
In this tutorial you will setup your own Mailserver running on an Ubuntu Server in the Hetzner Cloud.  
```

File: tutorials/setup-mailserver-with-mailcow/01.en.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    * You can read [this](https://help.github.com/en/enterprise/2.16/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) article to know how to generate SSH key
+* Write your server hostname in `name` input (mail.mydomain.com)
+* Click `Create & Buy Now`
+
+
+## Step 2 - Setup DNS
+
+Basicly you want to create a new DNS record called mail.domain.com and add your server IPv4 (A record) and IPv6 (AAAA record) to it.  
+Then you can setup your domains MX record to point to your cname that you just created.  
+You will also need to setup a autodiscover and autoconfig subdomain and both be a cname for mail.mydomain.com.
```
[the remark this answers is not in the document]
Comment (ntimo) [reply tutorials/setup-mailserver-with-mailcow/01.en.md]: You have to do that, some service need autodiscover for example if you want to use active sync on your iPhone you will need to have these subdomain setup. Also Mailcow will generate Let's Encrypt certs for them.

[the remark this answers is not in the document]
Comment (n-se) [reply tutorials/setup-mailserver-with-mailcow/01.en.md]: Okay, thanks for the information. Closing this as resolved.

File: tutorials/setup-mailserver-with-mailcow/01.en.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Basically you want to create a new DNS record called mail.example.com and add your server IPv4 (A record) and IPv6 (AAAA record) to it.  
+Then you can setup your domains MX record to point to your mail.example.com subdomain that you just created.  
+You will also need to setup a autodiscover.example.com and autoconfig.example.com subdomain both should be of the type CNAME and point to mail.example.com
+
+Your DNS configuration should look like this:
+
+```
+# Name              Type       Value
+mail                IN A       1.2.3.4
+mail                IN AAAA    1.2.3.4
```
Comment (n-se) [inline tutorials/setup-mailserver-with-mailcow/01.en.md]: You should use the default value `10.0.0.1` as stated in [contributing.md](https://github.com/hetzneronline/community-content/blob/master/contributing.md#terminology)

You use an IPv4 address and an AAAA record here? Better use the default IPv6 value `2001:db8:1234::1` here.

Comment (ntimo) [reply tutorials/setup-mailserver-with-mailcow/01.en.md]: True, didn't thought about that. Ups, will fix it.

**Comment by ntimo** [comment]
Thanks @n-se for the review :)

**Pull request merged**